spot-setup/Apache Spot Schema.md

```diff
@@ -0,0 +1,43 @@
+# Apache Spot Schema 
+
+This document is to centralize a place where users can read information about Proxy, DNS and Netflow schema.
```

Comment:
I'd add something to explain the users the motivation for this .md file. If some one wants to implement a module without using our ingest, they can do it by creating a data set with the expected columns by pipeline.